config/kernel-vfs-iov_iter.m4

```diff
@@ -201,4 +209,19 @@ AC_DEFUN([ZFS_AC_KERNEL_VFS_IOV_ITER], [
 		AC_DEFINE(HAVE_VFS_IOV_ITER, 1,
 		    [All required iov_iter interfaces are available])
 	])
+
+	dnl #
+	dnl # Kernel 6.5 introduces the iter_iov() function that returns the
```

Comment:
nit: It looks like this functionality was introduce in kernel 6.4. Anyway this comment could be updated to reflect that?

Reply:
Thanks, I've made the requested change, though am scratching my head a bit about why the existing code compiles fine on 6.4.4 (though I haven't dug very deeply on it, to be honest).